### PR TITLE
Updated strings from Crowdin 20220206

### PR DESCRIPTION
**Pull request by mikehardy**
Standard I18n sync

has some issues, opening it now but will need to work on context for cloze deletion at least

### CODE REVIEW

File: AnkiDroid/src/main/res/values-fa/02-strings.xml [hunk 1]
```diff
@@ -268,7 +268,7 @@
     <string name="choose_an_image">انتخاب تصویر</string>
     <!-- Card Template Browser Appearance -->
     <string name="restore_default">بازنشانی پیش‌فرض</string>
-    <string name="cloze_sample_text">This is a {{%1$s::sample}} cloze deletion.</string>
+    <string name="cloze_sample_text">Bonjour</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-fa/02-strings.xml:271]: this is not okay - cloze sample text probably needs context if the translator thought this was correct

**Review by mikehardy — COMMENTED** [review]
one issue with farsi cloze sample - needs fix on crowdin, maybe context

File: AnkiDroid/src/main/res/values-fa/02-strings.xml (hunk 1 above)
Comment (david-allison-1) [reply AnkiDroid/src/main/res/values-fa/02-strings.xml:271]: `fa` is farsi - presumed vandalism

File: AnkiDroid/src/main/res/values-uz/01-core.xml [hunk 2]
```diff
@@ -119,15 +119,15 @@
     <string name="menu_flag_card_five">Pushti bayroq</string>
     <string name="menu_flag_card_six">Feruza bayroq</string>
     <string name="menu_flag_card_seven">Siyohrang bayroq</string>
-    <string name="menu_edit_tags">Edit tags</string>
+    <string name="menu_edit_tags">Boy</string>
```
Comment (david-allison-1) [inline AnkiDroid/src/main/res/values-uz/01-core.xml:122]: 👀

**Comment by mikehardy** [comment]
Nice - thanks for the preening, much appreciated @david-allison-1

**Pull request merged**